docs/content/guides/6.best-practices/3.ai-contexts.md

```diff
@@ -0,0 +1,69 @@
+# Coding with AI
+
+Alokai aims to boost developer productivity in many ways. E.g. by giving you storefront accelerator and OOTB integrations with 3rd party services.
+Now we want to go even further by boosting your productivity with AI.
+
+## Disclaimer
+
+AI field is evolving rapidly. New models, new tools, new workflows are emerging every day.
+While we do our best to keep this guide updated, we might miss some things.
+Revisit this guide regularly and don't follow it blindly.
+
+## AI Contexts
+
+Alokai provides a set of AI contexts that can be used to code with AI.
+//TODO: explain what are AI contexts
+
+## Expectations Management
```

Comment:
Maybe I'd add that AI is made to work with it iteratively. I know people who think that it's one-off tool. You give it a task and if it failed, then they think AI is incapable of performing the task.